undertow/src/test/resources/org/wildfly/extension/undertow/undertow-7.1-transformers.xml

```diff
@@ -36,19 +36,17 @@
             <filter-ref name="limit-connections"/>
             <filter-ref name="headers" priority="${some.priority:10}"/>
             <filter-ref name="404-handler"/>
-            <!--
             <filter-ref name="static-gzip" predicate="path-suffix('.js')"/>
-            -->
          </location>
-         <!--<access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" predicate="not path-suffix(*.css)" prefix="access" rotate="false"/>-->
-         <access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" prefix="access" rotate="false"/>
+         <access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" predicate="not path-suffix(*.css)" prefix="access" rotate="false"/>
+         <!-- <access-log directory="${jboss.server.server.dir}" pattern="REQ %{i,test-header}" prefix="access" rotate="false"/> -->
```

Comment:
Can't this one simply be removed? The other line seems to contain everything and more

Reply:
This commit isn't mine, it's from #12363 so I don't want to change it.

Reply:
@kabir good catch, I'll submit a new PR to remote those, sorry for the mistake

Reply:
I mean, to remove those.